File: jhernandez9.c
//Johnny Hernandez
//CSC321
//Lab 9
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    int randInt, userGuess;
    srand(time(NULL));
    randInt = (rand() % 20) + 1;

    printf("Guess a number between 1 through 20:\n");
    scanf("%d", &userGuess);

    if(userGuess < 0 || userGuess > 20){
        printf("You input a number that is out of range (1-20) re-enter a valid number\n");
    }

    else if(userGuess == randInt){
        printf("you won and guessed the number correctly!\n");
    }
    else if(userGuess < randInt){
        printf("you lost and your guess is lower than my number which is %d\n", randInt);
    }
    else if(userGuess > randInt){
        printf("you lost and your guess is greater than my number which is %d\n", randInt);
    }
    return 0;
}

File: jhernandez9.java
//johnny hernandez
//CSC321
//lab 9
import java.util.Random;
import java.util.Scanner;

public class GuessingGame{
	public static void main(String[] args){
		Random rand = new Random();
		int randInt = rand.nextInt(1, 20);
		Scanner scanner = new Scanner(System.in);
		int attempts = 3;

		System.out.println(randInt);

		while(attempts > 0){
			System.out.println("\nGuess a number between 1 - 20");
			System.out.printf("You have %d attempts left\n", attempts);

			if(!scanner.hasNextInt()){
				System.out.println("Invalid input! Input must be an integer");
				scanner.next();
				continue;
			}

			int userGuess = scanner.nextInt();
			if(userGuess < 1 || userGuess > 20){
				System.out.println("Invalid input! Input must be in range of 1 - 20");
				continue;
			}

			if(userGuess == randInt){
				System.out.println("You Won!");
				return;
			}else if(userGuess < randInt){
				System.out.println("Too low!");
			} else {
				System.out.println("Too high!");
			}
			attempts--;
		}
		System.out.println("Loser");
	}
}
